pom.xml

```diff
@@ -78,7 +78,7 @@ under the License.
         <operator.sdk.version>4.8.3</operator.sdk.version>
         <operator.sdk.webhook-framework.version>1.1.1</operator.sdk.webhook-framework.version>
 
-        <fabric8.version>6.11.0</fabric8.version>
+        <fabric8.version>6.13.2</fabric8.version>
```

Comment:
Note that NOTICE file should be updated on dependency change, 
see: https://github.com/apache/flink-kubernetes-operator/blob/main/flink-kubernetes-operator/src/main/resources/META-INF/NOTICE

Reply:
Thanks for the heads up, fixed in a94b9ee6e88d0925b80501223f57db22fc9ff2ef